uwblueprint/headsupguys
# Auth reset pages UI

## Notion ticket link

[Reset Password](https://www.notion.so/uwblueprintexecs/Reset-Password-614cef5ae84846f7ac653bd6bf5a1398)



## Implementation description
* Reset password page has different stages that are the "inner" components of the page
* As you click through the screens, the different inner stages increment to change the inner part of the page
* The components (button-/components/button/index.tsx, checkbox- src/components/checkbox/index.tsx, textInput- src/components/textInput/index.tsx, and passwordInput- src/components/passwordInput/index.tsx) are part of another PR 



## Steps to test
1. Go to "/reset" and view page in mobile view. (Pages have been designed for mobile view so far) 
2. Can't move on from the "Reset Password" unless email is entered (note you need to click away from the input field
 after a valid email is entered)
![image](https://user-images.githubusercontent.com/43117838/124048793-1b148580-d9e5-11eb-873f-e311831e87d9.png)

3. Check Email page, the Open Email App button doesn't work yet (still investigating on if it's possible to directly open the email app from a webpage on mobile)
![image](https://user-images.githubusercontent.com/43117838/124048951-79d9ff00-d9e5-11eb-82db-567674c78b7e.png)

4. Password validation just checks if the two fields match and the password is more than 7 characters. 
5. Final screen has a button that redirects to login. 


## What should reviewers focus on?
* Only the /reset page 
* Just the overall look and feel of the screens in the reset password flow
* If anyone has input on how to directly open the mail app from the button on the 'Check Email' screen that'd be great  


## Checklist
